testsuite/testsuite.c

```diff
@@ -258,7 +258,7 @@ static inline int test_run_child(const struct test *t, int fdout[2], int fderr[2
 		}
 
 		if (stat_mstamp(&rootfsst) > stat_mstamp(&stampst)) {
-			ERR("rootfs %s is dirty, please run 'make rootfs' before running this test\n",
```

Comment:
I don't think these are equivalent. If I run `meson test`, it will actually run the test, which is not what I want. I think we need to expose a target like `ninja -C build create-rootfs`, so the rootfs is prepared for the test to run.

Reply:
And the right command would be....

```
meson compile testsuite/create-rootfs
```